test/concurrent_memcpy_test.cc

```diff
@@ -50,167 +50,168 @@ void testing_memcpy(const size_t size) {
 }
 
 void test_concurrent_memcpy_n_basic() {
-    std::vector<const void*> src_buffers;
```

Comment:
does v6d itself have any format tools? not in the CI? I am a little bit worry about the future rebase if we introduce format changes

Reply:
CI right now only does format check but the repo does have a formatting tool that can carry out potential format changes.

BTW, the reason that our previous commits have format issues is that we don't use the same version of clang-format as v6d CI does.

Reply:
Got you, make sense